splitgraph/ingestion/big_query/__init__.py

```diff
@@ -0,0 +1,127 @@
+import base64
+from typing import TYPE_CHECKING, Any, Dict, Optional
+
+from splitgraph.core.types import Credentials, Params, TableInfo
+from splitgraph.hooks.data_source.fdw import ForeignDataWrapperDataSource
+from splitgraph.ingestion.common import build_commandline_help
+
+if TYPE_CHECKING:
+    from splitgraph.engine.postgres.engine import PostgresEngine
+
+
+class BigQueryDataSource(ForeignDataWrapperDataSource):
+    credentials_schema: Dict[str, Any] = {
+        "type": "object",
+        "properties": {
+            "credentials_path": {
+                "type": "string",
+                "title": "Location of the GCP credentials",
+                "description": "The json credentials file need not be supplied when running inside a GCP VM",
+            },
+        },
+    }
+
+    params_schema = {
+        "type": "object",
+        "properties": {
+            "project": {
+                "type": "string",
+                "title": "GCP project name",
+                "description": "Name of the GCP project to use",
+            },
+            "dataset_name": {
+                "type": "string",
+                "title": "Big Query dataset",
```

Comment:
It's branded as BigQuery -- can you change it in the descriptions, as well as change the plugin name / package names to `bigquery` instead of `big_query`?

Reply:
Certainly, I was split about that as well.